jujugui/static/gui/src/app/components/icon-list/icon-list.js

```diff
@@ -0,0 +1,105 @@
+/* Copyright (C) 2018 Canonical Ltd. */
+'use strict';
+
+const PropTypes = require('prop-types');
+const React = require('react');
+
+/**
+  Display a list of icons for an entity.
+*/
+class IconList extends React.Component {
+  constructor() {
```

Comment:
The constructor usually receives props that must be propagated to super.
Why are we preventing that?

Reply:
Because nothing is being added in the constructor, you probably don't need to define one.